content/introduction-and-setup/walkthrough/cloning-a-project/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Cloning a C# Project"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 5
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Follow along with the provided walkthroughs to learn how to clone repos into Visual Studio. Try it out on your own by cloning the [LaunchCodeEducation/csharp-web-dev-datatypes](https://github.com/LaunchCodeEducation/csharp-web-dev-datatypes) repository page and fork the repository into your own GitHub account by selecting Fork from the top right of the page.
+
+## Windows Users
+
+[Clone a Git Repository in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/version-control/git-clone-repository?view=vs-2022)
+
+Note the path where you save this repo.
+
+## Mac Users
+
+[Cloning an Existing Repo](https://learn.microsoft.com/en-us/visualstudio/mac/set-up-git-repository?view=vsmac-2022#clone-an-existing-repository)
+
+Be sure to note the Target Folder where this repo is saved.
```

Comment:
This did not work for me.  Did I miss a step linking my github account to VS?  I do not have the option under code for my repo to open in visual studio.  

Under File:
Select Clone Repository
Pasted the copied ssh code in pop up url box
Pressed Clone button
Entered PassPhrase

Reply:
These are similar steps so I added a quick note that students can also find the option to clone under File. Hope this helps!

Reply:
My issue is that the documentation we are linking to for the Mac includes steps that I can not complete.  Is there a way to add the open with visual studio button in GitHub?  If not, can we add a bit more information that addresses this.

Reply:
I don't have one either so I added a note to let students know it is okay and how they can proceed forward.

Reply:
I updated the note to address the potential for that button not to appear for students. It isn't a cause for concern though.